rapid7_insightvm/bin/komand_rapid7_insightvm

```diff
@@ -4,10 +4,10 @@ import os
 import json
 from sys import argv
 
-Name = "Rapid7 InsightVM"
+Name = "Rapid7 InsightVM Console"
 Vendor = "rapid7"
-Version = "4.9.0"
-Description = "InsightVM is a powerful vulnerability management tool which finds, prioritizes, and remediates vulnerabilities. This plugin can get top remediations, scan results and start scans"
+Version = "5.0.0"
```

Comment:
@mrinehart-r7 With rename, is this just a patch bump?

Reply:
A plugin rename (only) is a patch 👍